helm-charts/splunk-otel-collector/templates/config/_common.tpl

```diff
@@ -151,12 +154,20 @@ resource/logs:
       action: delete
     - key: k8s.namespace.name
       action: delete
+    - key: k8s.pod.labels.app
+      action: delete
     {{- range $_, $label := .Values.extraAttributes.podLabels }}
     - key: {{ printf "k8s.pod.labels.%s" $label }}
       action: delete
     {{- end }}
     {{- end }}
     {{- end }}
+  
+transform/sck_fields:
+  log_statements:
+    - context: log
+      statements:
+        - set(resource.attributes["container_image"], Concat([resource.attributes["container.image.name"], resource.attributes["container.image.tag"]], ":"))
```

Comment:
This should not be added if  `.Values.splunkPlatform.fieldNameConvention.renameFieldsSck` is false

Reply:
Sorry about that, I earlier added that part in _otel_agent.tpl. I have updated the PR ;)